### DISCUSSION TITLE
Add SharePoint group as the owner of another SharePoint group

### DISCUSSION THREAD

**Discussion by dabaker6** [pnp/pnpcore#1316] [Ideas]
I have a need to add a SharePoint group as the owner of several other SharePoint groups, this is possible in the SharePoint UI and also in SharePointPnPCoreOnline, I can't find a method in PnPCore (unless I'm missing it!).

I've implemented it as a SOAP request in my code as there's no [SharePoint REST API endpoint](https://github.com/SharePoint/sp-dev-docs/issues/5031). Is this something of interest to be incorporated as a new method?

**Comment by jansenbe** [comment]
@dabaker6 : Great to see you've found a solution here and have written about it for others to learn from. We do have a model to support CSOM calls (which is what you do when using client.svc/ProcessQuery) in PnP Core SDK, I'm open for a PR that brings this feature to PnP Core SDK. See https://github.com/pnp/pnpcore/tree/dev/src/sdk/PnP.Core/Services/Core/CSOM for the CSOM code tree. The Requests folder in there contains the current implementations in PnP.Core. In PnP.Core.Admin there's also plenty of CSOM integrated: https://github.com/pnp/pnpcore/tree/dev/src/sdk/PnP.Core.Admin/Services/Core/CSOM.

When you check the code you'll notice we're not hardcoding the XML payloads as that allows us to batch these requests.

**Reply by dabaker6** [reply]
Great thanks, I've taken a quick look at the CSOM code in the PnP Core SDK. I may have some more questions when I come to implement it!
